libs/sdk-ui-ext/src/internal/components/configurationControls/ImageControl.tsx

```diff
@@ -0,0 +1,34 @@
+// (C) 2019-2024 GoodData Corporation
+import React from "react";
+import { useIntl } from "react-intl";
+import { ChartCellImageSizing } from "@gooddata/sdk-ui-charts";
+import DropdownControl from "./DropdownControl.js";
+
+import { imageDropdownItems } from "../../constants/dropdowns.js";
+import { getTranslatedDropdownItems } from "../../utils/translations.js";
+import { IVisualizationProperties } from "../../interfaces/Visualization.js";
+import { messages } from "../../../locales.js";
+
+export interface IImageControlProps {
+    pushData: (data: any) => any;
```

Comment:
Imho the correct type is `(data: IPushData) => void`. Many places use this black box type. I would rather change it here and in other places in this PR.